src/lib/__tests__/logger.test.js

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable global-require, import/no-dynamic-require, no-console */
+// Mock the console methods
+console.log = jest.fn();
+console.warn = jest.fn();
+console.error = jest.fn();
+
+const hasStackTraceRegex = /\/src\/lib\/logger\.js:\d+:\d+\)$/m;
+
+describe('logger', () => {
+  let logger;
+  let initializeLogger;
+  let updateDisplayAlert;
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    jest.resetModules();
+    ({ logger, initializeLogger, updateDisplayAlert } = require('../logger'));
+  });
+
+  it('throws when uninitialized', () => {
+    expect(logger.fatal).toThrow();
```

Comment:
For completeness use `toThrowError(...)`.

Reply:
And for the next 3 lines too.

Reply:
Because it's more readable you mean? I'm just asking because they are synonyms

Reply:
Are they synonyms :O? I thought toThrowError let's you specify exactly which error you expect. I just wanted you to fill in the specific expected error instead of any error.